sites/public/src/pages/index.tsx

```diff
@@ -190,15 +189,9 @@ export default function Home(props: IndexProps) {
   )
 }
 
-export async function getServerSideProps() {
-  const jurisdiction = await fetchJurisdictionByName(
-    runtimeConfig.getBackendApiBase(),
-    runtimeConfig.getJurisdictionName()
-  )
-
+export function getServerSideProps() {
```

Comment:
I'm not sure we need this fxn if we are no longer fetching data, and `locations` looks like static data